recipes/pthread-stubs/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix="$PREFIX"
+make
+make install
```

Comment:
This package only contains a `pkgconfig` file needed for `libxcb`, so the only test possible for it is checking that that file is properly installed (which I'm already doing :-)